### DISCUSSION TITLE
python -m onnxruntime.tools.make_dynamic_shape_fixed for multiple inputs

### DISCUSSION THREAD

**Discussion by frankxyy** [microsoft/onnxruntime#15151] [Other Q&A]
When I use this onnxruntime.tools.make_dynamic_shape_fixed api with muliple inputs like this: `python -m onnxruntime.tools.make_dynamic_shape_fixed --input_name input_ids --input_shape 1,512 --input_name bbox --input_shape 1,512,4 --input_name position_ids --input_shape 1,512 --input_name token_type_ids --input_shape 1,512 --input_name attention_mask --input_shape 1,512 --input_name image --input_shape 1,3,224,224 /home/xuyangyang/mseqa_surgeoned.onnx /home/xuyangyang/mseqa_static.onnx`, only the modification to the last input tensor in the command is done.

**Comment by wschin** [comment]
@skottmckay, do you have any idea? Many thanks.

**Comment by skottmckay** [comment]
It currently only supports updating one input name at a time. You can run it multiple times though.
